docs/design/proposals/adoption/resource-adoption.md

```diff
@@ -0,0 +1,201 @@
+        metadata:
+            name: data-bucket
+            namespace: team-a
+            annotations:
+                ...
+    aws:
+        nameOrID: my-data-bucket
+```
+
+**3. An adopted API Gateway v2 API (using an ARN)**
+
+```yaml
+apiVersion: apigateway.services.k8s.aws/v1alpha1
+kind: AdoptedResource
+metadata:
+    name: adopted-api-gateway
+spec:
+    kubernetes:
+        apiVersion: apigateway.services.k8s.aws/v1alpha1
+        kind: Api
+    aws:
+        arn: arn:aws:apigateway:us-east-1::/apis/123456789012
+```
+
+### Custom Resource Validation Webhook
+
+In order to minimise the chance of user’s facing unexpected behaviours, a validating webhook will validate any adopted resource changes prior to submission to the ACK resource managers. The webhook will primarily **reject** the following error cases:
+
+1. The provided target `GroupVersionKind` does not exist within the cluster
+2. The target resource would override an existing resource
+3. The target resource type cannot be adopted
+
+## Adopted K8s Resource Tagging
+
+All adopted resources will be annotated with an ACK-specific annotation that differentiates it from a natively created resource. This annotation serves to ensure clarity for users about possible specification differences introduced during the adoption process, or for filtering by adopted status. However, once adopted into a cluster, a resource will be reconciled with the same logic as any natively created resource. 
+
+The proposed k8s adopted tag is `services.k8s.aws/adopted: true`
+
+The adopted annotation would remain on the custom resource for the remainder of its lifecycle, and should not be modified by any other manager operation. 
+
+## AWS Resource Tagging
+
+AWS resources adopted by an ACK controller will have tags retroactively applied (whenever possible) to indicate its association with an ACK deployment. The details of these tags are discussed in another issue ([#148](https://github.com/aws/aws-controllers-k8s/issues/148)). ACK resource managers should not place any conditions on the existence of ACK tags on the AWS resources to gate the adoption process. The manager should should override any existing ACK tags.
+
+## Resource Type Blocklisting
+
+Adoption, for some resources, is not possible due to any number of the following reasons:
+
+* They are ephemeral resources which don’t support additional lifecycle operations (one-time jobs)
+* The service cannot provide adequate information to produce a full specification
+* Unforeseen limitations within ACK or the service directly
+
+For any of these cases, the resource can be marked as "not adoptable" and the validation webhook will subsequently reject any adopted resources of this type. Therefore, the manager will never attempt to run the adoption process. This option can be managed with a flag in the generator configuration file labelled `is_adoptable`.
+
+## Adoption Process Diagram
+
+![sequence-diagram](./images/adoption-sequence-diagram.png)
```

Comment:
remove the `/images` part of the above, since you placed the file into the `docs/design/propsals/adoption/` folder.

Reply:
I see you moved the png into an `images/` subdirectory, which is fine. Only problem is that the png file is an empty file according to Github :)

Reply:
It's showing as fine to me. 166 KB on the Github UI, and when viewing as raw I see the image.

Reply:
That is soooooo weird. It says "Empty file" for me in the pull request, but yes, I absolutely see it when I view it on your personal branch